styles.css

```diff
@@ -95,6 +140,11 @@ grid-area: 1/1/1/6;
     grid-area: 6/2/6/2;
```

Comment:
You don't need all these classes, considering it's a normal grid, it would be better to use code like this:

```css
.parent {
display: grid;
grid-template-columns: repeat(4, 1fr);
gap: 10px;
}
```

Reply:
yea I'll need help with that